mvc-framework/core/middlewares/cors/corsMiddleware.ts

```diff
@@ -30,16 +29,19 @@ interface MiddlewareData {
     useDefaultCors: boolean
 }
 
-export const handleCors = (requestContext: Mandarine.Types.RequestContext, data: MiddlewareData) => {
+export const handleCors = (requestContext: Mandarine.Types.RequestContext, data: MiddlewareData): boolean => {
     let { corsOptions, useDefaultCors } = data;
-    if(!corsOptions && useDefaultCors) corsOptions = Mandarine.Defaults.MandarineDefaultCorsOptions;
+    if (!corsOptions) {
+        return true;
+    }
+    if(useDefaultCors && !corsOptions.optionsSuccessStatus) corsOptions.optionsSuccessStatus = Mandarine.Defaults.MandarineDefaultCorsOptions.optionsSuccessStatus;
```

Comment:
Since we are going to use default values if cors is not present, then don't need this.